cardano-node/ChangeLog.md

```diff
@@ -651,9 +670,9 @@ No changes in the node. There were changes in the cardano-api and cardano-cli.
 ## 1.9.0 -- March 2020
 
 ### node changes
-- Remove CLI override for genesis file. It must be in the confg file. (#683)
-- Genesis file path in confg relative to the config file location (#648)
-- For security adjust default confg to not listen on local ports (#707)
+- Remove CLI override for genesis file. It must be in the config file. (#683)
+- Genesis file path in config relative to the config file location (#648)
```

Comment:
Can we confirm that #648 doesn't cause any issues with absolute paths?

Reply:
I didn't change anything here in the changelog except for spelling.